src/renderer/components/player-settings/player-settings.js

```diff
@@ -62,6 +62,12 @@ export default defineComponent({
       screenshotFolderPlaceholder: '',
       screenshotFilenameExample: '',
       screenshotDefaultPattern: '%Y%M%D-%H%N%S',
+      viewingModeValues: [
+        'default',
+        'theatre',
+        'fullwindow',
+        ...(process.env.IS_ELECTRON ? ['fullscreen', 'pip', 'external_player'] : []),
+      ]
```

Comment:
Probably best to make this a computed property, because it needs to contain the matching entries to the names but this currently doesn't check the external player setting, so they'll be out-of-sync if the external player is disabled.